javascripts/discourse/services/links-settings.js

```diff
@@ -39,11 +39,20 @@ export default class SocialLinksClickable extends Service {
       },
       {
         name: "X",
-        icon: "fab-twitter",
+        icon: "fab-x-twitter",
+        link: {
+          value: settings.x_custom_field_name,
+          base: "https://x.com/",
+          baseregex: "^http(s)?://(www.)?x.com/",
+        },
+      },
+      {
+        name: "Twitter",
+        icon: "fab-x-twitter",
```

Comment:
@bluefroguk1 Maybe revert this to `fab-twitter` to align with [this comment](https://github.com/weallwegot/discourse-social-links-clickable/pull/21#issuecomment-2661135614). Please also add it back to `settings.yml` in that case. Thanks!